### DISCUSSION TITLE
Add a warning for duplicated device name on different tenants

### DISCUSSION THREAD

**Discussion by 2code4me** [netbox-community/netbox#17827] [Ideas]
Hi,
I've noticed that there is a check for duplicated device name only limited to device related to the same tenant. 
If you use the same name for devices that are in different tenant there is no problem and you could create a device with same name and even same serial number. 

I understand well the logic behind this but I think could be usefull if, when you create a "duplicated" device, the system will show a warning message like:

"WARNING !! A device with the same name already exists in tenant [_tenant name_]. Would you like to continue?"
Yes / No

What do you think about it?

**Comment by AADIP** [comment]
Great use case for a custom validator. Theyre flexible enough that you would be able to do anything like the suggestion.
